docs/components/doc-management.md

```diff
@@ -1 +1,38 @@
-TBD
+This page details document management, including the ingestion and parsing of files, as well as searching through the files present in the Database.
+
+### Document Ingestion
+PDF files are parsed on a per-document, per-page basis. Once broken down into chunks (each of which is a page), a locally hosted model - `Alibaba-NLP/gte-base-en-v1.5` by default - generates vector embeddings for each chunk page. Each page/ chunk, with its metadata and embedding, is stored in the PostgreSQL database using vector indexing for efficient similarity search.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    User->>API: Upload file for ingestion
+    API->>DocumentService: Parse file into pages (chunks)
```

Comment:
hmmm... DocumentService to be is an internal abstraction so not sure if we should show it here. But it does allow us to show the chunking and embedding.

Also for embedding, we'd want to show an LLM or embedding model being call. I think I call it "LLM" in the chat docs.